includes/class-wc-install.php

```diff
@@ -520,7 +520,8 @@ private static function create_cron_jobs() {
 		$held_duration = get_option( 'woocommerce_hold_stock_minutes', '60' );
 
 		if ( '' !== $held_duration ) {
-			wp_schedule_single_event( time() + ( absint( $held_duration ) * 60 ), 'woocommerce_cancel_unpaid_orders' );
+			$cancel_unpaid_interval = apply_filters( 'woocommerce_cancel_unpaid_orders_interval_minutes', 5 );
```

Comment:
Hi @mfs-mindsize, thanks for the PR! Can we update this so that the scheduled event timing doesn't change when the filter isn't used? Instead of defaulting to `5`, I think it would be better here to use `absint( $held_duration )`.

Reply:
The current value of 60 feels rather large to me (given intent) but I'm not gonna go to the mat over it :) So the requested updates are in place. 

I'm curious, why also the absint()? There an absint() in the wp_schedule_single_event(). NBD, the extra absint() won't hurt. I'm just curious about your reasoning. (TIA)

Reply:
> The current value of 60 feels rather large to me (given intent) but I'm not gonna go to the mat over it :) So the requested updates are in place.

Thanks. I understand that concern, but I also didn't want the application of this change to affect anyone who expects the current behavior.

> I'm curious, why also the absint()? There an absint() in the wp_schedule_single_event(). NBD, the extra absint() won't hurt. I'm just curious about your reasoning. (TIA)

Oops, yeah, that's my fault. I really intended to only have it in the one spot and didn't consider that it was already in the `wp_schedule_single_event` call. Thanks for the change though, and sorry for the confusion.